data/changelogs/ddos-http.yaml

```
@@ -7,6 +7,7 @@ entries:
 - publish_date: '2024-03-12'
   scheduled: true
   individual_page: true
+  scheduled_date: "2024-04-15"
```

```suggestion
  scheduled_date: "2024-04-15"
  scheduled: true
  individual_page: true
```
Let's place the scheduled date right after the publish date, like in the WAF changelog.